packagers/jam.go

```diff
@@ -32,9 +33,21 @@ func (j Jam) WithExecutable(executable Executable) Jam {
 }
 
 func (j Jam) Execute(buildpackDir, output, version string, offline bool) error {
+
+	extensionTomlPath := filepath.Join(buildpackDir, "extension.toml")
+
+	buildpackOrExtensionToml := "buildpack.toml"
+	command := "--buildpack"
+	if _, err := os.Stat(extensionTomlPath); err == nil {
```

Comment:
I'd probably suggest using `packit/fs.Exists` here, as I think it's easier to reason about.